rpm/SPECS/contextBroker.spec

```diff
@@ -21,6 +21,14 @@
 %define name contextBroker
 %define owner orion 
 
+# To avoid problems in CentOS 8
```

Comment:
```
%if 0%{?rhel} > 7
# To avoid problems in CentOS 8
# What problems???
%define debug_package %{nil}
# To avoid /usr/lib/.build-id files in the package. This seems to be a
# new feature of CentOS 8 (see https://fedoraproject.org/wiki/Changes/ParallelInstallableDebuginfo)
# but by the moment we are going to disable it to build as in CentOS 7
%define _build_id_links none
%endif

```

Reply:
> What problems???

This ones:

https://wiki.mageia.org/en/Packaging_problems_and_solutions#Empty_Debug_Files

Reply:
We can include this link in the comment, which seems to be a precise description of the problem: https://www.programmersought.com/article/24984333483/

Reply:
Okis